tests/imap/test_folder_sync.py

```diff
@@ -93,7 +95,7 @@ def test_new_uids_synced_when_polling(db, generic_account, inbox_folder,
 
 
 def test_condstore_flags_refresh(db, default_account, all_mail_folder,
-                                 mock_imapclient, monkeypatch):
+                                 mock_imapclient, patched_gevent, monkeypatch):
```

Comment:
You can use `pytest.mark.usefixtures()` for this purpose - would love to see it used for all fixtures which the test does not need direct access to the fixture object to clean up test arguments: http://doc.pytest.org/en/latest/fixture.html#using-fixtures-from-classes-modules-or-projects

I think `mock_whatever` is clearer than `patched_whatever`, and would also love to see the terminology standardized. :)